packages/typegpu/src/std/index.ts

```diff
@@ -0,0 +1 @@
+export * from './numeric';
```

Comment:
I know its a lot, but lets list out every exported function, just so we can better control the public API.